stompserver/stompserver.go

```diff
@@ -103,15 +107,24 @@ type Lfnsite struct {
 	lfn  []string
 }
 
-// Metrics defines the metrics will be monitored using the HTTP server.
-type Metrics struct {
-	Received     uint64 `json:"received"`
-	Send         uint64 `json:"send"`
-	Traces       uint64 `json:"traces"`
-	Receivedperk uint64 `json:"Receivedperk"`
-}
+// prometheus metrics
+var (
+	Received = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "rucio_tracer_received",
```

Comment:
Before this goes in this should be corrected to `rucio_tracer_fwjr_received` and the rest of the metrics similarly. But Yuyi can do it too since the point of your PR is to show how to add the metrics.

Reply:
done, and you're right, I'm only showing you how to do it, but you can polish the code to whatever metrics you want.